federation/config.default.json

```diff
@@ -0,0 +1,83 @@
+[
+  {
```

Comment:
The schema is here - https://github.com/kubernetes/kubernetes-anywhere/blob/master/Kconfig. You need to follow the source-chain.

There is a discussion to move to a more explicit and simpler schema and/or config generation mechanism. See - https://github.com/kubernetes/kubernetes-anywhere/issues/128